source/includes/_lips_contacts.md

```diff
@@ -0,0 +1,109 @@
+### **Contacts**
+<a name="lips_contacts"></a>
+## Contacts API
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET `index` | /contacts? |
+
+### Usage
+Use GET to retrieve advertisers that match the query params.
+
+### Parameters
+When using the GET index method, the results will be filtered using these parameters:
+
+| Parameter | Required | Description |
+|---|---|---|
+
+<internal>
+|`event_params[recording_url]`|Yes|String that restricts the contacts to one or more based on recording_url|
+</internal>
+
+|`global_master_advertiser_id`|Yes|Restrict results to one or more specific gmaid|
+|`page_size`|No|Restrict number of keywords in result <br><b>Default value: 25</b> |
+|`page`|No|Specifies which page of results to return <br><b>Default value: 1</b>|
+
+<internal>
+#### If you use the event_params parameter, the response would contain also recording url and duration values
```

Comment:
If the `event_params` parameter is present, the response will also contain `recording_url` and `duration`.